diff --git a/kernel/Makefile b/kernel/Makefile
index 66929756..10cd0f99 100644
--- a/kernel/Makefile
+++ b/kernel/Makefile
@@ -123,9 +123,9 @@
 {\n\t\
 	struct cred *nonconst_cred = (struct cred *) cred;\n\t\
 	if (!cred)\n\t\t\
 		return NULL;\n\t\
-	if (!atomic_inc_not_zero(&nonconst_cred->usage))\n\t\t\
+	if (!atomic_long_inc_not_zero(&nonconst_cred->usage))\n\t\t\
 		return NULL;\n\t\
 	validate_creds(cred);\n\t\
 	return cred;\n\
 }\n
diff --git a/kernel/core_hook.c b/kernel/core_hook.c
index 9a8e774c..11dd3c39 100644
--- a/kernel/core_hook.c
+++ b/kernel/core_hook.c
@@ -593,30 +593,6 @@
 				pr_info("susfs: copy_to_user() failed\n");
 			return 0;
 		}
-		if (arg2 == CMD_SUSFS_HIDE_SUS_MNTS_FOR_ALL_PROCS) {
-			int error = 0;
-			if (arg3 != 0 && arg3 != 1) {
-				pr_err("susfs: CMD_SUSFS_HIDE_SUS_MNTS_FOR_ALL_PROCS -> arg3 can only be 0 or 1\n");
-				return 0;
-			}
-			susfs_hide_sus_mnts_for_all_procs = arg3;
-			pr_info("susfs: CMD_SUSFS_HIDE_SUS_MNTS_FOR_ALL_PROCS -> susfs_hide_sus_mnts_for_all_procs: %lu\n", arg3);
-			if (copy_to_user((void __user*)arg5, &error, sizeof(error)))
-				pr_info("susfs: copy_to_user() failed\n");
-			return 0;
-		}
-		if (arg2 == CMD_SUSFS_UMOUNT_FOR_ZYGOTE_ISO_SERVICE) {
-			int error = 0;
-			if (arg3 != 0 && arg3 != 1) {
-				pr_err("susfs: CMD_SUSFS_UMOUNT_FOR_ZYGOTE_ISO_SERVICE -> arg3 can only be 0 or 1\n");
-				return 0;
-			}
-			susfs_is_umount_for_zygote_iso_service_enabled = arg3;
-			pr_info("susfs: CMD_SUSFS_UMOUNT_FOR_ZYGOTE_ISO_SERVICE -> susfs_is_umount_for_zygote_iso_service_enabled: %lu\n", arg3);
-			if (copy_to_user((void __user*)arg5, &error, sizeof(error)))
-				pr_info("susfs: copy_to_user() failed\n");
-			return 0;
-		}
 #endif //#ifdef CONFIG_KSU_SUSFS_SUS_MOUNT
 #ifdef CONFIG_KSU_SUSFS_SUS_KSTAT
 		if (arg2 == CMD_SUSFS_ADD_SUS_KSTAT) {
